doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -111,8 +111,8 @@ this corresponds to working in the following context:
 Compatibility issues
 ~~~~~~~~~~~~~~~~~~~~
 
-Requiring the above modules creates an environment which is mostly
-compatible with the rest of Coq, up to a few discrepancies:
+Requiring the above modules creates an environment that is mostly
+compatible with the rest of Coq, up to a few discrepancies.
```

Comment:
The terminal colon is probably better since bullet points follow.  A period is OK, though.

Reply:
I understand that bullet points have no grammatical meaning, and just improve readability. Since what follows are sentences, a period seems to be preferable.